### DISCUSSION TITLE
esp32/Arduino/07.esp32-arduino-7segment

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#12] [General]
# esp32/Arduino/07.esp32-arduino-7segment

https://docs.geeksman.com/esp32/Arduino/07.esp32-arduino-7segment.html

**Comment by chinobi001** [comment]
有错误，在标题“如何判断是共阴还是共阳？”的第2种方法中，如果亮了说明是共阳型，反正还是共阳型······
